packages/components/text/src/text.styles.ts

```diff
@@ -56,15 +56,49 @@ const getTone = (tone: string) => {
 const getElementFontSize = (elementType?: string) => {
   switch (elementType) {
     case 'h1':
-      return '2.4615rem';
+      return `${designTokens.fontSizeForTextAsH1};`;
```

Comment:
Nit: you don't need string templates here.

```suggestion
      return designTokens.fontSizeForTextAsH1;
```